src/lib/openapi/spec/feature-environment-metrics-schema.ts

```diff
@@ -0,0 +1,34 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const featureEnvironmentMetricsSchema = {
```

Comment:
Could call this `featureMetricsDetailsSchema`, or even inline it into `featureMetricsSchema`.

Reply:
I didn't inline because it is also used in featureUsageSchema (lastHourUsage)